ferretdb_packaging/build_packages.sh

```diff
@@ -127,14 +130,14 @@ docker build --platform linux/amd64 -t documentdb-build-packages:latest -f ferre
     --build-arg BASE_IMAGE=$DOCKER_IMAGE --build-arg POSTGRES_VERSION=$PG --build-arg DOCUMENTDB_VERSION=$DOCUMENTDB_VERSION .
 
 # Run the Docker container to build the packages
-docker run --platform linux/amd64 --rm -v $abs_output_dir:/output documentdb-build-packages:latest
+docker run --platform linux/amd64 --rm --env OS=$OS_NAME -v $abs_output_dir:/output documentdb-build-packages:latest
```

Comment:
Let's use `--arg`/`ARG` instead of environment variables

Reply:
Looks like --arg is not available for `docker run`, only for `build`